<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="258" started-at="2019-01-26T09:24:18Z" finished-at="2019-01-26T09:24:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="258" started-at="2019-01-26T09:24:18Z" finished-at="2019-01-26T09:24:18Z">
      <class name="Tests.TestExecution">
        <test-method status="PASS" signature="object_repo()[pri:0, instance:Tests.TestExecution@19bb07ed]" name="object_repo" is-config="true" duration-ms="21" started-at="2019-01-26T14:54:18Z" finished-at="2019-01-26T14:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- object_repo -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Tests.TestExecution@19bb07ed]" name="beforeTest" is-config="true" duration-ms="0" started-at="2019-01-26T14:54:18Z" finished-at="2019-01-26T14:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="beforeclass()[pri:0, instance:Tests.TestExecution@19bb07ed]" name="beforeclass" is-config="true" duration-ms="206" started-at="2019-01-26T14:54:18Z" finished-at="2019-01-26T14:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeclass -->
        <test-method status="FAIL" signature="execute_tests(java.lang.String)[pri:0, instance:Tests.TestExecution@19bb07ed]" name="execute_tests" duration-ms="0" started-at="2019-01-26T14:54:18Z" data-provider="gettestcasename" finished-at="2019-01-26T14:54:18Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:143)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
	at Tests.TestExecution.gettestname(TestExecution.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:117)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- execute_tests -->
        <test-method status="PASS" signature="afterclass()[pri:0, instance:Tests.TestExecution@19bb07ed]" name="afterclass" is-config="true" duration-ms="2" started-at="2019-01-26T14:54:18Z" finished-at="2019-01-26T14:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Tests.TestExecution@19bb07ed]" name="afterTest" is-config="true" duration-ms="0" started-at="2019-01-26T14:54:18Z" finished-at="2019-01-26T14:54:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Tests.TestExecution -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
